atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -629,28 +667,20 @@ private ColumnSelection getColumnSelectionForRange(RangeRequest range) {
             }
         }
 
-        ColumnSelection columns = range.getColumnNames().isEmpty()
+        ColumnSelection cols = range.getColumnNames().isEmpty()
                 ? ColumnSelection.all() : ColumnSelection.create(range.getColumnNames());
 
-        ResultWithToken results = getTimestampsByCell(table, rows, columns, timestamp, batchSize, range.isReverse());
+        ResultWithToken result = getTimestampsByCell(table, rows, cols, timestamp, batchSize, range.isReverse(), token);
```

Comment:
This is a lot of parameters. Perhaps it would be good to do a refactor (in a future PR), extracting a method object or parameter object (or both). For now I think we should leave it as-is.